tools/release-prepare.sh

```diff
@@ -66,16 +67,17 @@ for file in "${files_to_change[@]}"; do
     sed -i "s/$prev_ver/$version/" "$file"
 done
 
-# Run `cargo check` to update firecracker and jailer versions in
-# `Cargo.lock`.
-say "Updating lockfile..."
+CHANGED=()
 cargo check
-CHANGED=(Cargo.lock)
 
-cd tests/integration_tests/security/demo_seccomp
-cargo check
-cd -
-CHANGED+=(tests/integration_tests/security/demo_seccomp/Cargo.lock)
+# Run `cargo check` to update firecracker and jailer versions in all
+# `Cargo.lock`.
+# NOTE: This will break if it finds paths with spaces in them
```

Comment:
Funnily enough, using the while loop instead of the for loop makes it work with spaces, so this comment isn't needed anymore 

```bash
> find . -path ./build -prune -o -name Cargo.lock -print |while read -r cargo_lock; do     echo "Updating $cargo_lock ..."; done
Updating ./tests/host_tools/uffd/Cargo.lock ...
Updating ./tests/integration_tests/security/demo_seccomp/Cargo.lock ...
Updating ./Cargo.lock ...
Updating ./little test/Cargo.lock ...
```